### PR TITLE
(feat#22) Partner Organization Query

### PR DESCRIPTION
**Pull request by santyssk**

### CODE REVIEW

**Comment by dickwyn** [comment]
@santyssk Include a description of your changes and in the PR title, follow what is in the CONTRIBUTING guide https://github.com/asucis/v3/blob/dev/CONTRIBUTING.md

File: src/pages/partnerorg.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <p>Data</p>
+      {data.allMarkdownRemark.edges.map(post => (
+        <div key={post.node.id}>
+          <p>{post.node.frontmatter.title}</p>
+          <p>{post.node.frontmatter.email}</p>
+          <p>{post.node.frontmatter.photo}</p>
+          <p>{post.node.frontmatter.link}</p>
+          <p>{post.node.frontmatter.description}</p>
+        </div>
+      ))}
```
Comment (dickwyn) [inline src/pages/partnerorg.js]: Is it possible to put this logic before returning the orgs.

So when a user calls this function, they can do data[0].title directly and not data.allMarkdownRemark.edges[0].node.frontmatter.title

Comment (santyssk) [reply src/pages/partnerorg.js]: Will check that

**Comment by santyssk** [comment]
feat(#22 ): Partner Organization Query

Added query for getting data of partner organizations in "src/hooks/get-partner-organization.js"
Created a new page for partner organizations "src/pages/partnerorg.js"
Added a link to partner organization from the first page in "src/pages/index.js"

File: src/pages/partnerorg.js (hunk 1 above)
Comment (dickwyn) [reply src/pages/partnerorg.js]: So the user can do something like that.

 const data = getPartnerOrganization();

<p>{data[0].title}</p>
<p>{data[0].email}</p>
<p>{data[0].photo}</p>
<p>{data[0].link}</p>
<p>{data[0].description}</p>

**Pull request merged**